Chef/CreateDevVm.ps1

```diff
@@ -351,7 +396,9 @@ function New-DevVm() {
     if ($global:devVmCreationWasSuccess -eq $true) {    
       if ($global:exportVm) {
         Stop-DevVm
+        Downgrade-DevVm-Resources
         Create-DevVm-Checkpoint
+        Rename-DevVm

```

Comment:
I think it makes sense to move Rename-DevVM before Create-DevVm-Checkpoint step.

Reply:
@calimetikcura The way the logic is setup we set the checkpoint name to "{DevVmName}-Created". Since we want the checkpoint name to be "RelativityDevVm-Created" it makes sense to have Rename after Create-Checkpoint. Unless we want the checkpoint name to be "RelativityDevVm-10.2.0.0-Created" etc.